assets/js/admin/settings/experimental-features.js

```diff
@@ -1,4 +1,7 @@
+import { on } from 'events';
```

Comment:
Didn't notice this auto-import. Removed here: https://github.com/Automattic/sensei/pull/7309/commits/f1fe5b7ffeb8b6fba361d7b2173a6dd8f5ae59d0